web/src/webflow/dropdown.html

```diff
@@ -0,0 +1,864 @@
+  }
+
+  #skf-dropdown-button {
+    width: 60px;
+    border-radius: 0 3px 3px 0;
+  }
+
+  #skf-combo-box-clear-button {
+    display: none;
+  }
+
+  #skf-dropdown-button:hover,
+  #skf-combo-box-clear-button:hover {
+    cursor: pointer;
+  }
+
+  #skf-dropdown {
+    border: none;
+    border-radius: 0 0 4px 4px;
+    margin: 0;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    max-height: 300px;
+    padding: 0;
+    position: absolute;
+    top: calc(1px + 100%);
+    left: 0;
+    width: 100%;
+    background-color: #fff;
+    box-shadow: 0px 2px 4px 0px #00000040;
+    z-index: 99;
+  }
+
+  #skf-dropdown a {
+    text-decoration: none;
+  }
+
+  #skf-dropdown li {
+    list-style: none;
+    margin-bottom: 4px;
+    padding: 18px 36px;
+    position: relative;
+  }
+
+  .skf-synonym {
+    background-color: #fff;
+  }
+
+  .skf-synonym p {
+    font-size: 14px;
+    font-style: italic;
+    font-weight: 300;
+    color: #757575;
+    margin: 10px 0;
+  }
+
+  .skf-synonym p span {
+    font-weight: 700;
+  }
+
+  .skf-industry:nth-child(even) {
+    background-color: #f9fbfb;
+  }
+
+  .skf-industry:hover {
+    background-color: #f5f5f5;
+    cursor: pointer;
+  }
+  .skf-industry-selected,
+  .skf-industry:nth-child(even).skf-industry-selected,
+  .skf-industry.skf-industry-selected:hover {
+    background-color: #e9eee0;
+  }
+
+  .skf-industry-selected::before {
+    content: url("data:image/svg+xml,%3Csvg viewBox='0 0 25 25' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M0 0h24v24H0z' fill='none'/%3E%3Cpath d='M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z'/%3E%3C/svg%3E");
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    left: 16px;
+    top: 24px;
+  }
+
+  .skf-industry-selected p {
+    font-weight: 600;
+  }
+
+  .skf-industry-name {
+    color: #1b1b1b;
+    font-weight: 400;
+    font-size: 16px;
+    margin-bottom: 4px;
+  }
+
+  .skf-industry-description {
+    color: #5c5c5c;
+    font-weight: 400;
+    font-size: 13px;
+    margin: 0;
+  }
+
+  #skf-combo-box-button {
+    background-color: transparent;
+    border: 2px solid white;
+    border-radius: 4px;
+    color: #fff;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    margin-left: 32px;
+    min-width: 217px;
+    text-align: center;
+    text-decoration: none;
+    display: none;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    height: 51px;
+  }
+
+  #skf-combo-box-button:hover {
+    background-color: #fff;
+    color: #3573b2;
+    cursor: pointer;
+  }
+
+  @media screen and (max-width: 700px) {
+    #skf-combo-box-container {
+      flex-direction: column;
+    }
+    #skf-combo-box {
+      max-width: unset;
+    }
+    #skf-combo-box-button {
+      margin: 20px auto 0;
+    }
+  }
+
+  @media screen and (max-width: 500px) {
+    #skf-combo-box-button {
+      margin: 20px 0 0;
+    }
+  }
+</style>
+<div id="skf-container" onclick="detectClickLocation(event)">
```

Comment:
[dust] If we're adding an `onclick` handler to a non-interactive component, there may be accessibility-related attributes we need to add like `role` so that it gets read out to a screen reader as interactive.